packages/create-toolpad-app/tests/index.spec.ts

```diff
@@ -58,6 +58,13 @@ test('create-toolpad-app can bootstrap a Toolpad app', async () => {
     }),
   );
 
+  // check that file exists or not in the directory
+  const gitignore = await fs.readFile(path.resolve(testDir, './.gitignore'), {
+    encoding: 'utf-8',
+  });
+
+  expect(gitignore.length).toBeGreaterThan(0);
+
   toolpadProcess = execa('yarn', ['dev'], {
```

Comment:
Could we add a test for the node version check as well? Run `nvm` inside the test to change Node versions and then check to see if the CLI stops executing as expected

Reply:
Run the specified version of nvm before we check whether the computer has this version or not. If there is no this version and it needs to be downloaded, then change the node version. It spends too long time in the process when running the test command.

Reply:
I'm ok with leaving this functionality untested for now if the only way to test it is to switch node versions. The overhead isn't worth it for this specific case imo.